### ISSUE TITLE
how to use lambda layer rebulid the input？

### ISSUE THREAD

**Issue by 1183710222** [yingkaisha/keras-unet-collection#53]
base_model = models.att_unet_2d((128, 128, 3), filter_num=[64, 128, 256, 512], n_labels=1, 
                           stack_num_down=2, stack_num_up=2, activation='ReLU', 
                           atten_activation='ReLU', attention='add', output_activation='Sigmoid', 
                           batch_norm=True, pool=False, unpool=False, 
                           backbone='VGG16', weights='imagenet', 
                           freeze_backbone=True, freeze_batch_norm=True, 
                           name='attunet')
                print(base_model.summary()) out put params =10M
                but I used the lambda layer and tried to  rebuild this shape
                like this
inp = Input(shape=(128, 128, N))
l1 = Conv2D(3, (1, 1))(inp) # map N channels data to 3 channels
out = base_model(l1)
model = Model(inp, out, name=base_model.name)
                 print(model.summary()) out put params = 6 
why？I sincerely seek for your assistance .
